src/libsync/configfile.cpp

```diff
@@ -863,7 +864,15 @@ void ConfigFile::setNewBigFolderSizeLimit(bool isChecked, qint64 mbytes)
 
 bool ConfigFile::confirmExternalStorage() const
 {
-    return getValue(confirmExternalStorageC, QString(), true).toBool();
+    const auto fallback = getValue(confirmExternalStorageC, QString(), true);
+    return getPolicySetting(QLatin1String(confirmExternalStorageC), fallback).toBool();
+}
+
+bool ConfigFile::useNewBigFolderSizeLimit() const
+{
+    const auto fallback = getValue(useNewBigFolderSizeLimitC, QString(), true);
+    const auto value = getPolicySetting(QLatin1String(useNewBigFolderSizeLimitC), fallback);
```

Comment:
Just one little thing please (I wonder if I forgot to mention it or if this was not addressed)

`return getPolicySetting(QLatin1String(useNewBigFolderSizeLimitC), fallback).toBool();` :)